src/cockpit/channel.py

```diff
@@ -95,6 +97,9 @@ def do_close(self):
     def do_pong(self, message):
         pass
 
+    def do_options(self, message):
+        raise ChannelError('This channel does not implement "options"')
```

Comment:
It needs a problem code as the first field.  Probably `protocol-error`?  What does the C bridge do here, anyway?

Reply:
Not, `not-supported`?